src/components/AddTodo/index.js

```diff
@@ -17,24 +24,33 @@ function AddTodo({ inputRef }) {
 	const dispatch = useDispatch();
 	const tasks = useSelector((state) => state.todos);
 
+	const { isLoading } = useSelector((state) => state.loading);
+
 	const handleSubmitTodo = (e) => {
 		if (e.keyCode === 13) {
 			if (valueInput.trim()) {
```

Comment:
!IMPORTANT: Do not check trim() function, in case `valueInput` is undefined => Red warning trim() is not a function

Reply:
Fixed!